internal/annotators/http/parser.go

```diff
@@ -0,0 +1,115 @@
+package http
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+)
+
+func RemoveExtraSpaces(s string) string {
+	return strings.Join(strings.Fields(s), " ")
+}
+
+func HTTPParser(r *http.Request) string {
+
+	//Signature Inputs extraction
+	var rgx = regexp.MustCompile(`\((.*?)\)`)
+	fmt.Printf("r: %v\n", r)
+	signatureInput := r.Header.Get("Signature-Input")
+	fmt.Printf("signatureInput: %v\n", signatureInput)
```

Comment:
What is the plan for these `fmt.Printf` statements? Are these temporary to facilitate console logging in development? Do you foresee the need to introduce logging support as a feature within the annotators?

Reply:
All the printf statements are temporary to facilitate console logging in development, the logging support will not be a feature within the annotators